client/apps/edit/components/admin/test/index.coffee

```diff
@@ -99,3 +103,22 @@ describe 'EditAdmin', ->
       @view.$('input[name=credit_url]').val().should.equal 'http://credit'
       @view.$('.edit-email-small-image-url input').val().should.containEql 'img.png'
       @view.$('.edit-email-large-image-url input').val().should.containEql 'img.png'
+
+  describe '#toggleScheduled', ->
+
+    it 'can save the scheduled publish date and time', ->
+      @view.article.set {scheduled_publish_at: null}
+      date = '11/11/11'
+      time = '12:30'
+      dateAndTime = date + ' ' + time
+      global.confirm = -> true
+      @view.$('.edit-admin-input-date').val(date)
+      @view.$('.edit-admin-input-time').val(time)
+      @view.toggleScheduled(@e)
+      @view.article.get('scheduled_publish_at').should.containEql moment(dateAndTime, 'MM/DD/YYYY HH:mm').toDate()
+      delete global.confirm
```

Comment:
Doesn't seem like it'll be a big problem for Positron, but be careful when dealing with globals like this b/c it can affect other tests. Generally best to use sinon.stub & restore in beforeEach and afterEach blocks

e.g.

````coffeescript
beforeEach: ->
  sinon.stub global, 'confirm'

afterEach: ->
  global.confirm.restore()
````

Although I realize confirm might not exist at all in this case :-P I think [this array in benv](https://github.com/artsy/benv/blob/master/index.js#L6) probably needs more attention to conventionalize things.